### ISSUE TITLE
Focussable nav elements have no visual indicator

### ISSUE THREAD

**Issue by nternetinspired** [madebykind/shu-design-system-base#9]
Some keyboard focussable navigation elements have their default outline removed without a replacement being supplied, failing WCAG requirement 2.4.7. to provide a visual indicator, as described in https://www.w3.org/WAI/WCAG21/Techniques/failures/F78
